lib/_nonstd.scm

```diff
@@ -3098,7 +3109,7 @@
            (let* ((prefix
                    (or path
                        (##path-expand
-                        "gsx-temp"
+                        (##string-append (##process-mnemonic-identifier) "-temp")
                         (##os-path-tempdir))))
                   (pid
```

Comment:
The adding of the PID should no longer be done by `##create-temporary-directory`.

Reply:
Ok; where we used to see "`-temp`" at the end of the temporary directory's name, we'll now see "`-pid`" (please let me know If I've misunderstood).

Reply:
I was referring to line 3114 which gets the pid and later uses it, which is no longer necessary. Line 3112 is OK, so the general pattern for the directory name is `<executable-name>-<pid>-temp`.